php/swoole/start.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Swoole\Http\Server;
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+
+$server = new Server('0.0.0.0', 3000);
+$server->set([
+    'daemonize'  => false,
+    'worker_num' => swoole_cpu_num() * 2,
```

Comment:
please use the same config as existing (made by @sy-records), I need to find an easiest way to include them
```suggestion
   'worker_num' => swoole_cpu_num() * 2,
    'enable_coroutine' => false,
    'log_file' => '/dev/null',
    'log_level' => SWOOLE_LOG_ERROR,
```